src/runtime_src/core/tools/common/ReportCmcStatus.cpp

```diff
@@ -47,9 +48,10 @@ ReportCmcStatus::getPropertyTree20202( const xrt_core::device * _pDevice,
   }
   catch(const xrt_core::query::no_such_key&) {}
   catch(const xrt_core::query::sysfs_error&) {}
+
+  boost::property_tree::ptree pts;

```

Comment:
We have extra characters available, please make your variables a little bit more description.